modules/microshift-config-yaml.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-config-yaml_{context}"]
+= Using YAML configuration
+
+Configure your {product-title} using a YAML file. Values specified in your configuration file are used when you execute a command.
+// Q: can these values be overriden with CLI flags?
+
+== Default Settings
+If you do not create a config.yaml, the following default values will be used.
+// Q: what creates this default yaml?
+// Q: can it be modified? (instead of making a new one)
+
+.Default YAML file example
+[source,yaml]
+----
+cluster:
+  clusterCIDR: 10.42.0.0/16
+  serviceCIDR: 10.43.0.0/16
+  serviceNodePortRange: 30000-32767
+  dns: 10.43.0.10
+  domain: cluster.local
+  url: https://127.0.0.1:6443
+nodeIP: ""
+nodeName: ""
+logVLevel: 0
+----
+
+IMPORTANT
+The configuration file must be located at `~/.microshift/config.yaml` (user-specific) and `/etc/microshift/config.yaml` (system-wide). An existing user-specific `config.yaml` takes precedence.
+
+.Procedure
+//Q: can this be done with a command, or is this an instance where user goes into the directory and creates a text file?
+
+Create the YAML file with the following formatting:
+
+.Formatted YAML file example
+[source,yaml]
+----
+cluster: ""
+  clusterCIDR: ""
+  serviceCIDR: ""
+  serviceNodePortRange: ""
+  dns: ""
+  domain: ""
+  url: ""
+nodeIP: ""
+nodeName: ""
+logVLevel: ""
+----
+//Q: does the user need to enter every value? will any default?
```

Comment:
All of them should have defaults. There's a sample file that should have the current defaults at https://github.com/openshift/microshift/blob/main/packaging/microshift/config.yaml